common/models/dataset.js

```diff
@@ -678,4 +678,14 @@ module.exports = function(Dataset) {
       logger.logError(err.message, { location: "Dataset.metadatakeys" });
     }
   };
+
+  Dataset.logbook = async function (id, filters = "{}") {
+    const Logbook = app.models.Logbook;
+
+    // returns logbook associated with dataset
+    const ds = await Dataset.findById(id);
+
+    // the logbook chat has the same name as the proposal id
+    return Logbook.findDatasetLogbook(ds.proposalId, filters);
```

Comment:
shouldn't the logbook have a proposalId = the relation to the proposal?

Reply:
Unfortunately, logbook does not have a relation to proposal. 
Currently , here at ESS, the logbook created for a specific proposal is named with the proposal id. That's the only way the two are related.
It would be a nice development if the logbook will have the proposal id. I guess that we need to do more brainstorming on this feature to generalize it.

Reply:
are you suggesting waiting for such a discussion or we can merge and keep it for later?

Reply:
I'm suggesting to merge now and brainstorm later on how to improve it